string name
float64 h_1
float64 h_2
float64 h_3
float64 e_term1
float64 e_term2
float64 e_term3
float64 v_1
float64 v_2
float64 v_3
float64 v_squared_1
float64 v_squared_2
float64 v_squared_3
float64 v_cubed_1
float64 v_cubed_2
float64 v_cubed_3
float64 v_exp_1
float64 v_exp_2
float64 v_exp_3
float64 loss_v_term
float64 loss_v_squared_term
float64 loss_volt_term
float64 loss_volt_squared_term
float64 loss_resistance_12v
float64 loss_resistance_0v
float64 voltage_exponent
float64 volt_squared_current 
float64 current_squared_volt 
float64 custom_v_c 
float64 custom_c_pow 
float64 custom_v_pow 
float64[] initial_temps
